mxabierto/sara
# 73 links

Solves #73 
# How to test
+ Recreate indexes in elasticsearch
`python create_recommendations.py`
+ Send a GET request to recommendations
`echo  "Http://0.0.0.0:5000/recommendations?title=quiero chamba"|sed 's/ /%20/g'|xargs curl -X GET`
+ Verify that the answer contains an array in "links"
